docs/src/reference/docbook/reference/shell.xml

```diff
@@ -295,6 +295,11 @@ public class HelloWorldCommands implements CommandMarker {
 	<literal>org.springframework.shell.commands</literal> package as well as the section in this documentation
 	of Built in commands.</para>
       </listitem>
+
+      <listitem>
+        <para><literal>--disableColor</literal> - Flag that disables colorization.  There is no argument to this option.
```

Comment:
might be worth following Git's pattern and calling this `porcelain` mode: https://git-scm.com/docs/git-status